basic-javascript/project-2.spec.js

```diff
@@ -1,3 +1,142 @@
 const funcs = require('./project-2');
 
-// whoops.. there is no test suite for this file. You'll simply just have to create one :/
+describe("getBiggest", () => {
+    it("should get the biggest number", () => {
+        expect(funcs.getBiggest(9, 5)).toBe(9);
+        expect(funcs.getBiggest(3, 3)).toBe(3);
+        expect(funcs.getBiggest(3, 8)).toBe(8);
+    });
+});
+
+describe("greeting", () => {
+    it("should get a proper greeting depending on the language", () => {
+        expect(funcs.greeting()).toBe("Hello!");
+        expect(funcs.greeting("German")).toBe("Guten Tag!");
+        expect(funcs.greeting("Spanish")).toBe("Hola!");
+    });
+});
+
+describe("isTenOrFive", () => {
+    it("should return true if the num is 10 or 5", () => {
+        expect(funcs.isTenOrFive(7)).toBeFalsy();
+        expect(funcs.isTenOrFive(5)).toBeTruthy();
+        expect(funcs.isTenOrFive(10)).toBeTruthy();
+    });
+});
+
+describe("isInRange", () => {
+    it("should check if the num is in range between 20 and 50", () => {
+        expect(funcs.isInRange(3)).toBeFalsy();
+        expect(funcs.isInRange(24)).toBeTruthy();
+    });
+});
```

Comment:
A good base case would be to test 20,50 returns false
Another if 49, 21 returns true?
With corresponding messages.